code/src/index.css

```diff
@@ -11,3 +11,158 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/*MOVIE LIST*/
+.movie-list {
+  display: flex;
+  flex-wrap: wrap;
+  background-color: black;
+}
+
+.movie-cards {
+  position: relative;
+}
+
+.text-overlay {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.text-overlay h1 {
+  position: absolute;
+  top: 100;
+  bottom: 0;
+  padding-bottom: 50px;
+  margin-left: 20px;
+}
+
+.text-overlay p {
+  position: absolute;
+  top: 100;
+  bottom: 0;
+  margin-left: 20px;
+  margin-bottom: 40px;
+}
+
+.movie-overlay {
+  max-width: 400px;
+}
+
+.movie-image {
+  max-width: 400px;
+}
+
+.movie-text {
+  color: #fff;
+  display: none;
+}
+
+/*HOVER-EFFECT*/
+
+.text-overlay:hover {
+  background-color: #000000a1;
+}
+
+.text-overlay:hover .movie-text {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+}
+
+/*MOVIE DETAILS*/
+
+.backdrop-image {
+  min-height: 200px;
+  background-size: cover;
+}
+
+.back-button {
+  position: absolute;
+  display: inline-flex;
+  color: #fff;
+  align-items: center;
+  font-weight: 700;
+  text-decoration: none;
+  left: 50px;
+  top: 30px;
+}
+
+.back-button svg {
+  width: 30px;
+}
+
+.movie-details {
+  position: relative;
+}
+
+.image-and-text {
+  display: flex;
+  flex-direction: column;
+}
+
+.small-image {
+  border: 5px solid #fff;
+}
+
+.rating {
+  color: red;
+  font-size: 17px;
+  font-weight: 400;
+  margin-left: 10px;
+}
+
+.movie-info {
+  max-width: fit-content
+}
+
+.details {
+  height: fit-content;
+  background-color: #000000a1;
+  padding: 10px;
+}
+
+.details h1 {
+  color: #fff;
+}
+
+.details p {
+  color: #fff;
+}
+
+
+@media (min-width: 1024px) {
+
+.movie-list a {
+    width: 25%;
+}
+
+.backdrop-image {
+  min-height: 100vh;
+}
+
+.image-and-text {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  padding-top: 380px;
+}
+
+.details {
+  margin-left: 20px;
+  width: 400px;
+}
+
+.small-image {
+  margin-left: 50px;
+}
+
+.movie-overlay {
+  max-width: 500px;
+}
+
+.movie-image {
+  max-width: 500px;
+}
```

Comment:
I think this part is one of the reason why your page on desktop is scrollable ( it has like a sidebar in the bottom of the page) And i think that needs to be fixed. 
I tried somethings on the devtools when i was looking at your page and if i on movie-img have width 100% instead  the scroll disappears and all of your movies are displayed in it's right size ( 25%). Right now your first 3 movies has one size and the 4th one is larger.